app/models/sample.rb

```diff
@@ -856,4 +872,13 @@ def metadata_fields_info
   def first_pipeline_run
     pipeline_runs.order(created_at: :desc).first
   end
+
+  def status_url
+    base_url = if Rails.env == 'staging'
```

Comment:
You did not find a way to get rid of the domain here? In theory, the rails app could move to any domain.
I still think it should at least move to a generic helper/concern/config.

Reply:
No I didn't do that yet. I'll look into it.

Reply:
Updated